packages/app/cypress/e2e/runner/support/verify-failures.ts

```diff
@@ -151,7 +152,7 @@ const verifyFailure = (options) => {
     if (uncaught) {
       cy.log('uncaught error has an associated log for the original error')
       cy.get('.command-name-uncaught-exception')
-      .should('have.length', 1)
+      .should(mode === 'component' ? 'have.length.gte' : 'have.length', 1)
```

Comment:
Uncaught errors in CT are caught & reported multiple times

Reply:
Can we log a follow up issue and link it to the GA epic?

Reply:
Looks like you already did!